### ISSUE TITLE
Feature Request: Change attributes of modded mobs

### ISSUE THREAD

**Issue by IchHabeHunger54** [p1ut0nium-git/Rough-Mobs-Revamped#31]
Hi,

I was testing with some 1.15 and i didn't quite like the max health of some modded entities. Now I was wondering if you could add this into the mod, as I think it would perfectly fit.

Thanks!

**Comment by p1ut0nium-git** [comment]
Have you tried adding a mod entity to the list with: modid:entity;attribute;etc

**Comment by IchHabeHunger54** [comment]
Yes i have (specifically with "quark:frog;generic.maxHealth;0;-6")

Also, I noted that it doesn't work with some vanilla entities as well (e. g. bees or rabbits).
On zombies and skeletons it works fine, though.

**Comment by p1ut0nium-git** [comment]
Ah. That's because it only works with Monster types, no passives. If the entity extends from the MonsterEntity class it will work. I may consider changing it though. I'll have to look int it.

**Comment by IchHabeHunger54** [comment]
I'd love to be able to change every living entity's attributes, not just monster ones. (Personally because I think bees having 10 HP is way too much, as chicken only have 4). Thanks!
